packages/platforms/ingrasys/x86-64/x86-64-ingrasys-s9100/platform-config/r0/src/python/x86_64_ingrasys_s9100_r0/__init__.py

```diff
@@ -0,0 +1,496 @@
+from onl.platform.base import *
+from onl.platform.ingrasys import *
+import os
+
+class OnlPlatform_x86_64_ingrasys_s9100_r0(OnlPlatformIngrasys):
+    PLATFORM='x86-64-ingrasys-s9100-r0'
+    MODEL="s9100"
+    SYS_OBJECT_ID=".8.1"
```

Comment:
@SamCC-Chang  Please note that the SYS_OBJECT_ID value must be unique across your platforms. 
(This one looks like the result of copy paste from a different platform). 

The SYS_OBJECT_ID here is joined with your Private Enterprise Number. 

I would suggest you change this so something meaningful like ".9100". 
Whatever you set it to you will need a system in place to allocate them uniquely per platform.